File: Sistema de Control Clínico Veterinario/Sistema_Control_Clinico_Veterinario/logica/Persona.cs
using System;

namespace SistemaVeterinario.logica
{
    public abstract class Persona
    {
        public int Id { get; set; }
        public string Nombre { get; set; }
        public string Dpi { get; set; }
        public string Telefono { get; set; }
        public string Email { get; set; }
        public string Direccion { get; set; }
        
        public string Usuario { get; set; }
        public string Contrasena { get; set; }

        public override string ToString() => Nombre;
    }
}

File: Sistema de Control Clínico Veterinario/bluej/Persona.java
public class Persona {
    private String nombre;
    private String dpi;
    private String telefono;
    private String email;
    private String direccion;
    
    public Persona(String nombre, String dpi, String telefono, String email) {
        this.nombre = nombre;
        this.dpi = dpi;
        this.telefono = telefono;
        this.email = email;
    }
    
    public String getNombre() { return nombre; }
    public void setNombre(String nombre) { this.nombre = nombre; }
    public String getDpi() { return dpi; }
    public String getTelefono() { return telefono; }
    public String getEmail() { return email; }
}